juego.py

```diff
@@ -181,14 +201,13 @@ def play(self, level):
 
         another_quest = True
 
-        right_sound = load_sound("./data/right.ogg")
-        wrong_sound = load_sound("./data/wrong.ogg")
-        fondo = [load_image("data/" + str(x) + ".jpg") for x in range(0,NUM_IMAGES)]
-        random_index = random.randint(0,NUM_IMAGES - 1)
+        right_sound = load_sound("right.ogg")
+        wrong_sound = load_sound("wrong.ogg")
+        fondo = load_image(str(1) + ".jpg")
```

Comment:
Yes, that is a possible way to do it. However, the code was written to accomodate more than one background images. I would like to keep that style. If in the future we decide to add more than one image, it would be easy to adapt by replacing the 1 with a variable.